packages/backend-modules/mailchimp/lib/scheduler/archiveUnsubscribed.ts

```diff
@@ -23,12 +23,12 @@ module.exports = async (dryRun = false) => {
     MAILCHIMP_PROBELESEN_AUDIENCE_ID,
   ]
 
-  audiencesToArchiveUnsubscribed.forEach((audienceId) => {
-    archiveUnsubscribed({ dryRun, audienceId })
-  })
+  bluebird.each(audiencesToArchiveUnsubscribed, async (audienceId) =>
+    archiveUnsubscribedInAudience({ dryRun, audienceId }),
+  )
 }
 
-const archiveUnsubscribed = async ({ dryRun, audienceId }) => {
+const archiveUnsubscribedInAudience = async ({ dryRun, audienceId }) => {
   const mailchimp = MailchimpInterface({ console })
   const unsubscribedMembers = await mailchimp.getMembersFromAudienceWithStatus(
     MailchimpInterface.MemberStatus.Unsubscribed,
```

Comment:
why throw an error if !unsubscribedMembers (not a new change). Shouldn't it only throw an error if mailchimp.getMembersFromAudienceWithStatus actually throws an error?

Reply:
It doesn't throw an error, it logs an error in case unsubscribedMembers is null or undefined which can be helpful for debugging. That usually happens when the mailchimp API returns an error